techniques/fileDistribution/copyGitFile/2.2/copyFileFromSharedFolder.st

```diff
@@ -52,6 +52,10 @@ bundle agent download_from_shared_folder
       &COPYFILE_DESTINATION:{destination |"copyfile[&i&][destination]" string => "&destination&";
 }&
 
+      # Raw version for reporting
+      &COPYFILE_DESTINATION:{destination |"copyfile[&i&][raw_destination]" string => "&destination&";
+}&
```

Comment:
shouldn't you change l66 to "canon_dirname_${index}" string => canonify(dirname("${copyfile[${index}][raw_destination]}")); ?

Reply:
actually, no, that's correct. Thank you